Documentation/5.4.1/README.rst

```diff
@@ -1568,6 +1645,43 @@ ironic-inspector.yaml for Ironic-Inspector Deployment
       IronicInspectorExtraProcessingHooks: extra_hardware,lldp_basic,local_link_connection,nuage_lldp
 
 
+ovs-hw-offload.yaml
+~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    # A Heat environment file that enables OVS Hardware Offload in the overcloud.
+    # This works by configuring SR-IOV NIC with switchdev and OVS Hardware Offload on
+    # compute nodes. The feature supported in OVS 2.8.0
```

Comment:
Do we need to mention the OVS version here. Our VRS is OVS 2.5 based . I understand relevant code has been backported from upstream OVS 2.8 but still our base OVS remains the same.

Reply:
@shivsg, this is an environment file that comes with openstack-tripleo-heat-templates package itself.